app/components/funding_widget_component.html.erb

```diff
@@ -30,7 +30,7 @@
             <h3><%= sub_head %></h3>
             <div>
               <p>
-                A scholarship or bursary isn't available for your chosen subject, but you can still get a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a> if you're eligible (non-UK citizens without settled status are unlikely to be eligible).
+                A scholarship or bursary isn't available for your chosen subject, but you can still get a <a href="/funding-and-support/tuition-fee-and-maintenance-loans">tuition fee and maintenance loan</a> if you're eligible (non-UK citizens without <a href="https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk">indefinite leave to remain in the UK</a> are unlikely to be eligible).
```

Comment:
@EllieNodder Why is there a link in this default widget option but not in the other subject-specific ones?

Unless you feel it is critical here, I would suggest removing the link as a) there are already lots of links in the widget text so it is distracting and b) we've already linked to this in the intro section above the widget so we are essentially repeating the link twice with the same link text which is not great in terms of accessibility best practice